### PR TITLE
5064 translations for report print / export

### LINKED ISSUE #5064: Translations for report print not working
## Is your feature request related to a problem? Please describe 👀
Currently, we have report [translations for UI](https://github.com/msupply-foundation/open-msupply/issues/4677), but no translations when the user prints or exports to a PDF or excel. 

## Describe the solution you'd like 🎁
Make sure print report translations are working

### Moneyworks Jobcode 🧰
OMS:REPT

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #5064, #4995

# 👩🏻‍💻 What does this PR do?
Passes in the current language to the print api and also fixes wrapping of status chips in `expiring items` and `stock status` reports.

# 🧪 Testing

1.
- [ ] Change the language of the app
- [ ] Go to a report and try print
- [ ] The report should be in the language that you have selected
2. 
- [ ] Go to either `expiring items` or `stock status` report
- [ ] Change the size of the browser to be small
- [ ] The chips shouldn't wrap anymore

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/system/src/Report/api/hooks/usePrintReport.ts [hunk 1]
```diff
@@ -81,6 +83,7 @@ export const usePrintReport = () => {
       format,
       arguments: args,
       sort,
+      currentLanguage,
```
Comment (roxy-dao) [inline client/packages/system/src/Report/api/hooks/usePrintReport.ts:86]: Actually fix to get language through to printing / pdf

File: server/graphql/reports/src/print.rs [hunk 2]
```diff
@@ -118,7 +118,7 @@ pub async fn generate_report(
         report_data,
         arguments,
         format.map(PrintFormat::to_domain),
-        translation_service,
+        *translation_service,
```
Comment (roxy-dao) [inline server/graphql/reports/src/print.rs:121]: Sidequests...

File: server/reports/expiring-items/2_3_0/src/style.css [hunk 3]
```diff
@@ -52,6 +52,7 @@
 .container table td.expiry-date span.expired {
   background-color: rgba(245, 5, 1, 0.3);
   padding: 3px 10px;
+  white-space: nowrap;
```
Comment (roxy-dao) [inline server/reports/expiring-items/2_3_0/src/style.css:55]: Fix for chips wrapping

**Review by andreievg — CHANGES_REQUESTED** [review]
Looks great, but I am actually not a big fan of cloning the translation service, can we access it via reference ?

**Comment by roxy-dao** [comment]
> Looks great, but I am actually not a big fan of cloning the translation service, can we access it via reference ?

Not cloning translation service? 🤔

**Comment by andreievg** [comment]
> > Looks great, but I am actually not a big fan of cloning the translation service, can we access it via reference ?
> 
> Not cloning translation service? 🤔

I meant like this one, sorry should have been more specific 

https://github.com/msupply-foundation/open-msupply/compare/5064-Translations-for-report-print-/-export...5064-Translations-for-report-print-/-export(Avoid-cloning-translations-service)

**Review by andreievg — APPROVED** [review]
Thanks @roxy-dao, looks great

**Pull request merged**